en-US/Conventions_for_Writers_and_Editors.xml

```diff
@@ -27,7 +27,7 @@
 		<title>Usage Dictionary</title>
 		 <partintro>
 			<para>
-				The Usage Dictionary provides guidelines for the correct use of common terms in Red&nbsp;Hat documentation, which terms to avoid, and the preferred spelling if variations exist.
+				The Usage Dictionary provides guidelines for the correct use of common terms in Red&#160;Hat documentation, which terms to avoid, and the preferred spelling if variations exist.
```

Comment:
Why replace nbsp with hex? .. isn't that more difficult to remember?

Reply:
Because for some unknown reason, since F32 `&nbsp;` doesn't work right there. It works everywhere else, works in RHEL, haven't tested in F33. It's only this one instance so I wasn't too concerned about it.